src/metrocs/redistricting/StarGeneratorTest.java

```diff
@@ -0,0 +1,19 @@
+package metrocs.redistricting;
+import static org.junit.jupiter.api.Assertions.assertNotNull;
+import org.junit.jupiter.api.Test;
+import java.util.ArrayList;
+/**
+ * Tests for Voter objects.
+ *
+ * @author Jada, David, Colin, John, Daniel, Ali
+ * @version
+ */
+public class StarGeneratorTest {
+    @Test
+    public void generatePatternTest() {
+        //check if method returns null
```

Comment:
This is a vacuous comment in that it doesn't add any additional explanation to the code that isn't obvious by reading the code itself. The information delivered by this comment should be **why** this is checking whether or not the method returns null.